course-v2/layouts/partials/course_info.html

```diff
@@ -0,0 +1,104 @@
+            </div>
+            <div class="course-info-content {{ if $inPanel }}panel-course-info-text{{ end }}">
+              {{ partial "partial_collapse_list.html" (dict "list" $instructors "id" "instructors" "key" "title" "klass" "course-info-instructor" "showCollapse" $inPanel) }}
+            </div>
+          </div>
+        {{ end }}
+
+        {{ with $courseData.department_numbers }}
+          <div class="mt-4">
+            {{ $departments := partial "get_departments.html" . }}
+            <div class="{{ if $inPanel }}panel-course-info-title {{ else }} text-muted {{ end }}">
+              Departments
+            </div>
+            <div class="course-info-content {{ if $inPanel }}panel-course-info-text{{ end }}">
+              {{ partial "partial_collapse_list.html" (dict "list" $departments "id" "departments" "key" "title" "klass" "course-info-department" "useLinks" true) }}
+            </div>
+          </div>
+        {{ end }}
+
+        {{ if $inPanel }}
+          <div class="mt-4">
+            <div class="{{ if $inPanel }}panel-course-info-title {{ else }} text-muted {{ end }}">
+              As Taught In
+            </div>
+            <div class="course-info-content {{ if $inPanel }}panel-course-info-text{{ end }}">
+              {{ $courseData.term }} {{ if isset $courseData "year" }}{{ $courseData.year }}{{ end }}
+            </div>
+          </div>
+
+          <div class="mt-4">
+            <div class="{{ if $inPanel }}panel-course-info-title {{ else }} text-muted {{ end }}">
+              Level
+            </div>
+            <div class="course-info-content {{ if $inPanel }}panel-course-info-text{{ end }}">
+              {{ if eq (printf "%T" $courseData.level) "string" }}
+                {{ $levelSearchUrl := partial "get_search_url.html" (dict "key" "level" "value" $courseData.level) }}
+                <a href="{{ $levelSearchUrl }}" class="course-info-level">
+                  {{ $courseData.level }}
+                </a>
+              {{ else }}
+                {{ range $courseData.level }}
+                  {{ $levelSearchUrl := partial "get_search_url.html" (dict "key" "level" "value" .) }}
+                  <a href="{{ $levelSearchUrl }}" class="course-info-level">
+                    {{ . }}
+                  </a><br />
+                {{ end }}
+              {{ end }}
+            </div>
+          </div>
+        {{ end }}
+      </div>
+      
+      {{ if not $inPanel }}
+      <div class="col-12 col-sm-6">
+        <div class="mt-4 mt-sm-0">
+          <div class="text-muted">
+            Topics
+          </div>
+          <div class="course-info-content">
+            {{ partial "topics_oneline.html" . }}
+          </div>
+        </div>
+      </div>
+      {{ end }}
+    </div>
+  </div>
+</div>
+
+<script>
+// todo: move this function to js file.
+// currently having some issues while using this function from js file
+function desktopCourseInfoToggle(){
```

Comment:
I'm making a follow-up PR for https://github.com/mitodl/ocw-hugo-themes/issues/795 and will change this approach along with moving this code to a suitable place.